UnitTests/CatalogTests/ServiceTests/CommentsServiceTest/AddCommentTest.cs

```diff
@@ -0,0 +1,77 @@
+﻿using Catalog.Services.Comments;
+using Common.Interfaces;
+using Common.Models.Comments;
+using FluentValidation;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+using System.Threading.Tasks;
+
+namespace UnitTests.CatalogTests.ServiceTests.CommentsServiceTest
```

Comment:
Please take a look at the existing unit test to make similar folders and namespaces for the sake of the code convention consistency. These 'Tests' ending in each part of the namespace take a lot of space in the Test Explorer and therefore making the testing process less user-friendly.

Example from the tests of 'ProductMerger.MergeProduct' method's namespace:
```
namespace UnitTests.Catalog.Services.ProductMerger
{
    [TestClass]
    public class MergeProductTests
    {
```

- [ ] Make changes to the tests folder and namespaces to make them shorter and easier to use among the developers.